EnglishFilter/sections/whitelist.txt

```diff
@@ -1611,11 +1611,6 @@ rawstory.com#@#div[id^="div-gpt-ad"]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/36491
 @@||js-sec.indexww.com/ht/htw-baltimore-sun.js$domain=baltimoresun.com
 @@||c.amazon-adsystem.com/aax2/apstag.js$domain=baltimoresun.com
-! https://github.com/AdguardTeam/AdguardFilters/issues/36197

```

Comment:
@BlazDT Please check if these rules are required(added in your commit https://github.com/AdguardTeam/AdguardFilters/commit/67ce3415f32749e12e613c842003728d387c38a2)